cli/lsp/language_server.rs

```diff
@@ -324,6 +324,59 @@ impl Inner {
     Ok(navigation_tree)
   }
 
+  fn merge_user_tsconfig(
+    &mut self,
+    maybe_config: &Option<String>,
+    maybe_root_uri: &Option<Url>,
+    tsconfig: &mut TsConfig,
+  ) -> Result<(), AnyError> {
+    self.maybe_config_file = None;
+    self.maybe_config_uri = None;
+    if let Some(config_str) = maybe_config {
+      if !config_str.is_empty() {
+        info!("Updating TypeScript configuration from: \"{}\"", config_str);
+        let config_url = if let Ok(url) = Url::from_file_path(config_str) {
+          Ok(url)
+        } else if let Some(root_uri) = maybe_root_uri {
+          let root_path = root_uri
+            .to_file_path()
+            .map_err(|_| anyhow!("Bad root_uri: {}", root_uri))?;
+          let config_path = root_path.join(config_str);
+          Url::from_file_path(config_path).map_err(|_| {
+            anyhow!("Bad file path for configuration file: \"{}\"", config_str)
+          })
+        } else {
+          Err(anyhow!(
+            "The path to the configuration file (\"{}\") is not resolvable.",
+            config_str
+          ))
+        }?;
+        info!("  Resolved configuration file: \"{}\"", config_url);
```

Comment:
```suggestion
        info!("Resolved configuration file: \"{}\"", config_url);
```

Reply:
The spacing is intentional, as it occurs after the value is read. The spacing was already there, it was just the code refactored to handle errors better.